W-20617379-pce-4-backup-restore-influxDB-kt (#393)
mulesoft/docs-private-cloud: modules/ROOT/pages/backup-and-disaster-recovery.adoc

The backup file is stored on the NFS storage.

== Create an InfluxDB Backup

> To create an InfluxDB backup, run the following command:

Reviewer: ```suggestion
To create an InfluxDB backup, run the this command:
```

`curl -k <platform-dns>/platform/amv/backup-X POST -H 'Authorization: Bearer ${token}' -H 'Content-Type: application/json' -d '{"backup-file-name": "${backupFileName}.tar.gz", "nfs-server": "${fileSystemDns}", "nfs-path": "/"}'`

== Perform an InfluxDB Restore

> To perform an InfluxDB restore, run the following command:

Reviewer: ```suggestion
To perform an InfluxDB restore, run this command:
```
